src/main/1.3/scala/com/holdenkarau/spark/testing/JavaStreamingSuitebase.scala

```diff
@@ -16,67 +16,43 @@
  */
 package com.holdenkarau.spark.testing
 
-import org.apache.spark.streaming._
-import org.apache.spark._
-import org.apache.spark.SparkContext._
-
-import java.io._
 import java.util.{List => JList}
 
-import scala.collection.mutable.ArrayBuffer
-import scala.collection.mutable.SynchronizedBuffer
+import org.apache.spark.api.java.function.{Function => JFunction, Function2 => JFunction2}
+import org.apache.spark.streaming.api.java._
+import org.apache.spark.streaming.dstream.DStream
+import org.junit.Assert._
+
 import scala.collection.JavaConversions._
 import scala.collection.immutable.{HashBag => Bag}
 import scala.language.implicitConversions
 import scala.reflect.ClassTag
 
-import org.apache.spark.streaming.dstream.{DStream, InputDStream}
-import org.apache.spark.streaming.api.java._
-import org.apache.spark.api.java.function.{Function => JFunction}
-import org.apache.spark.streaming.scheduler.{StreamingListenerBatchStarted, StreamingListenerBatchCompleted, StreamingListener}
-import org.apache.spark.streaming.util.TestManualClock
-import org.apache.spark.{SparkConf, Logging}
-import org.apache.spark.rdd.RDD
-
-import org.junit._
-import org.junit.Assert._
-
 /**
-  * This is the base trait for Spark Streaming testsuites. This provides basic functionality
+  * This is the base trait for Spark Streaming testsuite. This provides basic functionality
   * to run user-defined set of input on user-defined stream operations, and verify the output.
- * This implementation is designer to work with JUnit for java users
- * Note: this always uses the manual clock
- */
+  * This implementation is designer to work with JUnit for java users.
+  *
+  * Note: this always uses the manual clock
+  */
 class JavaStreamingSuiteBase extends JavaSuiteBase with StreamingSuiteCommon {
 
   override def conf = super.conf
     .set("spark.streaming.clock", "org.apache.spark.streaming.util.TestManualClock")
 
-  override def runStreams[V: ClassTag](
-    outputStream: TestOutputStream[V],
-    ssc: TestStreamingContext,
-    numBatches: Int,
-    numExpectedOutput: Int
-    ): Seq[Seq[V]] = {
-    val output = super.runStreams(outputStream, ssc, numBatches, numExpectedOutput)
-    assertEquals("Unexpected number of outputs generated", numExpectedOutput, output.size)
-    output
-  }
-
   /**
    * Verify whether the output values after running a DStream operation
    * is same as the expected output values, by comparing the output
    * collections either as lists (order matters) or sets (order does not matter)
    */
-  def verifyOutput[V: ClassTag](
-    output: Seq[Seq[V]],
-    expectedOutput: Seq[Seq[V]],
-    useSet: Boolean
-  ) {
+  def verifyOutput[V: ClassTag](output: Seq[Seq[V]],
+                                expectedOutput: Seq[Seq[V]],
```

Comment:
We should probably stick with the Spark style guide even though its different than the auto indent (4 spaces for multi-line arguments like this).